test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+# frozen_string_literal: false
+
+require_relative '../helper'
+
+require 'csv'
+
+class TestFilter < Test::Unit::TestCase
+
+  # Some rows data (useful as default).
+  Rows = [
+    %w[aaa bbb ccc],
+    %w[ddd eee fff],
+  ]
+
+  def setup
+    # In case the previous test left this as true.
+    $TEST_DEBUG = false
+  end
+
+  # Print debugging information if indicated.
+  def debug(label, value, newline: false)
+    return unless $TEST_DEBUG
+    print("\n") if newline
+    printf("%15s: %s\n", label, value.inspect)
+  end
+
+  # Return the test name (retrieved from the call stack).
+  def get_test_name
+    caller.each do |x|
+      method_name = x.split(' ').last.gsub(/\W/, '')
+      return method_name if method_name.start_with?('test')
+    end
+    raise RuntimeError.new('No test method name found.')
+  end
```

Comment:
We can use `Test::Unit::TestCase#name`, `Test::Unit::TestCase#local_name` or `Test::Unit::TestCase#method_name`.